### ISSUE TITLE
DeltaFlow model weight

### ISSUE THREAD

**Issue by Kevin-C-0904** [KTH-RPL/OpenSceneFlow#26]
Hi!
We highly appreciate your work in the field of flow estimation. We noticed that DeltaFlow ranks first on the AV2 leaderboard and are wondering if you could provide the corresponding best checkpoint. This would allow us to compare the performance differences between Flow4D and DeltaFlow on the AV2 validation set. We observed that the pretrained weights of DeltaFlow provided in the codebase are for 5-frame input with a 38m range, which is inconsistent with the range used by Flow4D.
Thanks!

**Comment by Kin-Zhang** [comment]
Hi

Please check the implementation detail in our paper where we pointed out
the best cpkt trained hyperparameters etc and the provided ckpt is the one
we used in the leaderboard(also the best one)

Different method have their different best params, if you want to keep
hyperparameters same feel free to retrain it use what params you want. For
local results in the paper we retrain our and other methods under the same
parameters (again please check paper implementation detail section to know
more)
